csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -1768,6 +2013,10 @@ void InnerOuterPersistentKernelScheduler::schedule(
       rparams != nullptr && rparams->scheduler_type == schedulerType(),
       "Incorrect parameters sent to InnerOuterPersistentKernelScheduler::schedule",
       params);
-  scheduleInnerOuterPersistentKernel(fusion, rparams);
+  if (rparams->tma_warp_specialized) {
+    scheduleInnerOuterWarpSpecializedTmaKernel(fusion, rparams);
```

Comment:
`scheduleInnerOuterWarpSpecializedTmaKernel` is a duplication of `scheduleInnerOuterPersistentKernel`, will be modified in following PRs.